src/InstagramClone.Api/Controllers/UsersController.cs

```diff
@@ -36,5 +36,29 @@ public IActionResult GetUser(Guid id)
             return Ok(user);
         }
 
+
+
+        [HttpPost("")]
+        public async Task<IActionResult> PostUsers([FromBody] UserCreateDTO userDto)
```

Comment:
Why `async` when you aren't using any `await` in the method?

Ideally you should only use async with await. So in this case, if you change it to remove the async, you also have to remove the Task<> from the return type.